### DISCUSSION TITLE
TraceContext in MQTT message

### DISCUSSION THREAD

**Discussion by ennc0d3** [emqx/emqx#14359] [General]
I've enabled opentelemetry tracing in Monitoring, Does EMQX include the tracecontext on the message's UserProperties?

**Comment by thalesmg** [comment]
Yes, see the docs: https://docs.emqx.com/en/emqx/latest/observability/open-telemetry/traces.html#integrate-opentelemetry-tracing

Since only MQTTv5 supports `User-Properties`, only this protocol will forward the context.
